backend/apps/sandbox/tests_reviews.py

```
@@ -42,3 +42,43 @@ def test_code_review_edge_cases():
+
+    # 1. Create session via POST
+    response = api_client.post("/api/sandbox/collab-sessions/", {"project": str(project.id)})
+    assert response.status_code == 201
+    session_id = response.data["id"]
+
+    # 2. Retrieve session
+    response = api_client.get(f"/api/sandbox/collab-sessions/{session_id}/")
+    assert response.status_code == 200
+
+    # 3. Create another user to join the session
+    another_user = User.objects.create_user(username="guest_user", password="guestpassword123")
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to argument: "password"

(S106)